tests/run-make/min-global-align/rmake.rs

```diff
@@ -0,0 +1,27 @@
+// This test checks that global variables respect the target minimum alignment.
+// The three bools `STATIC_BOOL`, `STATIC_MUT_BOOL`, and `CONST_BOOL` all have
+// type-alignment of 1, but some targets require greater global alignment.
+// See https://github.com/rust-lang/rust/pull/44440
+
+//@ only-linux
+// Reason: this test is target-independent, considering compilation is targeted
+// towards linux architectures only.
```

Comment:
Nit: did you mean "target-dependenent"?

Reply:
I meant target independent as in, "since we compile to certain targets, what OS we run this test on probably doesn't matter, so the test is independent of OS", but that might be wrong and certainly was unclear. Edited.